mappings/47689e21e411e9acfbeb967bd50127757f03a293206f3cd9e4ced23c545253.json

```diff
@@ -0,0 +1,54 @@
+{
+    "subject": "47689e21e411e9acfbeb967bd50127757f03a293206f3cd9e4ced23c545253",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "",
+        "signatures": [
+            {
+                "signature": "041cd8a36c0697ff0b5695da51e3448546b25996b18ba7a118ac87d013424a01e12a755ca433b2368a6170aaf6dc4bfb7e3a41c3d269a340d1265660db2cd509",
```

Comment:
@siddigsalahaldin,

you removed the value, but the signature is the same. Url field can't be submitted empty, otherwise, the validator sees it as not a valid url.
I suggest you completely remove the "url" values and submit again.